ote/src/cljs/ote/views/own_services.cljs

```diff
@@ -236,35 +236,22 @@
      [table-container-for-own-services e! has-services? operator-services state]]]])
 
 
+(defn no-operator-texts
+  [e! state]
+  [:div {:style {:margin "3rem 0"}}
+   [:p (tr [:own-services-page :own-services-no-providers])]
+   [:a (merge {:href "#/transport-operator"
```

Comment:
merging seems unnecessary because following works as well:
`     [:a {:href "www.testi.fi"
          :on-click #(do
                       (.preventDefault %)
                       (e! (to/->CreateTransportOperator)))}
      "testilinkki"]`

Reply:
I think that stylefy returns a map with classname or something and it needs to be merged to map that contains those href and on-click attributes